book/src/terminology.md

```diff
@@ -50,6 +50,10 @@ with a ledger interpretation that matches the leader's.
 
 A [ledger](#ledger) derived from common entries but then diverged.
 
+#### fortnight (placeholder name, could be "turn", "term?")
+
+The time (some number of [voting periods](#voting period)) during which a [leader](#leader) ingests transactions and produces [entries](#entry).
```

Comment:
This what I was calling a slot.

Reply:
yeah, and #1990 makes a new recommendation...

Reply:
ok, do we want a one-word description of a "voting period"?  if so, how about "chunk", or "slice"? 

if we use "term" here, we can use "congress", or "ballot"